ledger/acctupdates.go

```diff
@@ -1140,6 +1143,9 @@ func (au *accountUpdates) lookupWithoutRewards(rnd basics.Round, addr basics.Add
 		// a separate transaction here, and directly use a prepared SQL query
 		// against the database.
 		persistedData, err = au.accountsq.lookup(addr)
+		if err != nil {
+			return ledgercore.AccountData{}, basics.Round(0), rewardsVersion, rewardsLevel, err
```

Comment:
By convention, shouldn't we return 0s for the version and level too?

Reply:
I thought about this but below we return non-zeros for whatever reason. I'll check the error code checking in callers and update if they are fine